torchvision/ops/feature_pyramid_network.py

```diff
@@ -184,6 +184,7 @@ def __init__(self, in_channels, out_channels):
         self.use_P5 = in_channels == out_channels
 
     def forward(self, p, c, names):
+        # type: (List[Tensor], List[Tensor], List[str])
```

Comment:
The lint failure is related - you need to return the type as well, similar to the other pooler
```python
# type: (List[Tensor], List[Tensor], List[str]) -> Tuple[List[Tensor], List[str]]
```